src/Sample.Extension/Sample.Extension.csproj

```diff
@@ -11,6 +11,8 @@
     <AssemblyName>Sample.Extension</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
Yeah good point. Not sure what happened there. I've reverted all changes in this folder.